da/avail/avail.go

```diff
@@ -0,0 +1,227 @@
+package avail
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+	"strings"
+
+	ds "github.com/ipfs/go-datastore"
+	"github.com/rollkit/rollkit/da"
+	"github.com/rollkit/rollkit/da/avail/datasubmit"
+	"github.com/rollkit/rollkit/log"
+	"github.com/rollkit/rollkit/types"
+)
+
+type Config struct {
+	BaseURL    string  `json:"base_url"`
+	Seed       string  `json:"seed"`
+	ApiURL     string  `json:"api_url"`
+	AppID      int     `json:"app_id"`
+	confidence float64 `json:"confidence"`
+}
+
+type DataAvailabilityLayerClient struct {
+	namespace types.NamespaceID
+	config    Config
+	logger    log.Logger
```

Comment:
Kept these fields as private. If so, make sure you don't use these fields outside this package.

Reply:
yes,these fields are private.We are not using this outside of this package